fusor-ember-cli/app/mirage/config.js

```diff
@@ -199,72 +187,70 @@ export default function() {
+                           "Intel Westmere Family",
+                           "Intel SandyBridge Family",
+                           "Intel Haswell-noTSX Family"],
+            default_family: "Intel Haswell-noTSX Family"
+           };
+  });
 
-    this.put('/contacts/:id');
-    this.put('/contacts/:id', 'user'); // specify the type of resource to be updated
-  */
+  this.get('/customer_portal/owners/:owner_key/consumers', function(db, request) {
+    return db.management_applications;
+  });
 
-  /*
-    DELETE shorthands
+  this.get('/customer_portal/consumers/:uuid/entitlements', function(db, request) {
+    return db.entitlements;
+  });
 
-    this.del('/contacts/:id');
-    this.del('/contacts/:id', 'user'); // specify the type of resource to be deleted
+  this.get('/customer_portal/users/:username/owners', function(db, request) {
+    return db.owners;
+  });
 
-    // Single object + related resources. Make sure parent resource is first.
-    this.del('/contacts/:id', ['contact', 'addresses']);
-  */
+  this.get('/customer_portal/pools', function(db, request) {
+    return db.pools;
+  });
 
-  /*
-    Function fallback. Manipulate data in the db via
+  this.post('/customer_portal/login', function(db, request) {
+    return {};
+  });
 
-      - db.{collection}
-      - db.{collection}.find(id)
-      - db.{collection}.where(query)
-      - db.{collection}.update(target, attrs)
-      - db.{collection}.remove(target)
+  this.post('/customer_portal/login', function(db, request) {
+    return {};
+  });
+  this.post('/customer_portal/consumers/:uuid/entitlements', function(db, request) {
+    return {};
+  });
 
-    // Example: return a single object with related models
-    this.get('/contacts/:id', function(db, request) {
-      var contactId = +request.params.id;
+  // route to prevent js console error by ember-cli-blanket
+  this.post('/write-blanket-coverage', function(db, request) {
+    return {};
+  });
 
-      return {
-        contact: db.contacts.find(contactId),
-        addresses: db.addresses.where({contact_id: contactId})
-      };
-    });
+  this.put('fusor/api/v21/deployments/:id/deploy', function(db, request) {
+    // fusor/server saves foreman_task_uuid to deployment
+    // but mirage doesn't have access to ember-data store,
+    // so mirage can't save the foreman_task_uuid to the deployment and
+    // transition to review/progress/overview
+    // fusor/server returns deployment
+    return db.foreman_tasks.find('db25a76f-e344-48ba-ac77-f29303586dbe');
+  });
 
-  */
-}
+  this.get('/api/v21/foreman_tasks');
+  this.get('/api/v21/foreman_tasks/:id', function(db, request) {
+    var id = request.params.id;
+    return db.foreman_tasks.find(id);
+  });
 
-/*
-You can optionally export a config that is only loaded during tests
-export function testConfig() {
+  // All route NOT defined in mirage/config.js
+  // will passthrough to real backend server
+  this.passthrough();
```

Comment:
this line `this.passthrough()` means that all routes NOT declared in by mirage will pass through to the proxy server. 

```
ember s --environment partial-mocks-enabled --proxy http://sat62devg-qci.example.com
```